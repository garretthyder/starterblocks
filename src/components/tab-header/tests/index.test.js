import { shallow, mount } from 'enzyme';
import {TabHeader} from '../';

describe('Error Notice', () => {
    // 1. Snapshot testing
    it('renders correctly', () => {
        const setSearchContext = jest.fn();
        const setActiveItemType = jest.fn();
        const wrapper = shallow(<TabHeader setSearchContext={setSearchContext} setActiveItemType={setActiveItemType} />);
        expect(wrapper).toMatchSnapshot();
    });


});
