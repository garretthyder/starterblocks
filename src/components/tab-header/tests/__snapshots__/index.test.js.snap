// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Notice renders correctly 1`] = `
<div
  className="starterblocks-builder-modal-header"
>
  <div
    className="template-search-box"
  >
    <div>
      <i
        className="fas fa-search"
      />
      <input
        className="form-control"
        onChange={[Function]}
        placeholder="Type to search"
        type="search"
      />
    </div>
  </div>
  <div
    className="starterblocks-template-list-header"
  >
    <button
      className=""
      onClick={[Function]}
    >
       
      Sections
       
    </button>
    <button
      className=""
      onClick={[Function]}
    >
       
      Pages
       
    </button>
    <button
      className=""
      onClick={[Function]}
    >
       
      Collections
       
    </button>
    <button
      className=""
      onClick={[Function]}
    >
       
      Saved
       
    </button>
    <button
      className="starterblocks-builder-close-modal"
      onClick={[Function]}
    >
      <i
        className="fas fa-times"
      />
    </button>
  </div>
</div>
`;
